### ISSUE TITLE
Not calling setServer means begin() fails

### ISSUE THREAD

**Issue by buelowp** [mcxiaoke/ESPDateTime#7]
Using version 0.2.0, if I do not call setServer with a valid server, the begin call causes isTimeValid() to return false and time is not set.

```
    DateTime.setServer("pool.ntp.org");
    
    DateTime.begin();

    if (!DateTime.isTimeValid()) {
        Serial.println("Failed to get time from server.");
    }
```

This will always work

```
//    DateTime.setServer("pool.ntp.org");
    
    DateTime.begin();

    if (!DateTime.isTimeValid()) {
        Serial.println("Failed to get time from server.");
    }
```

This will always fail after the default 10 second timeout.

**Comment by fusionstream** [comment]
I have the same issue but is slightly different. Using sg.pool.ntp.org always fails, not setting a server always fails, but setting to asia.pool.ntp.org sometimes works and sometimes doesn't. Increasing timeout seems to make no difference.

**Comment by 877dev** [comment]
Same here, it seems adding the line fixes the issue, thanks!
`
  DateTime.setServer("pool.ntp.org"); `

**Comment by mcxiaoke** [comment]
default server changed, may fixed in v0.2.1
